cli/vm/cli.go

```diff
@@ -640,12 +647,22 @@ func handleSlots(c *cli.Context) error {
 // prepareVM retrieves --historic flag from context (if set) and resets app state
 // (to the specified historic height if given).
 func prepareVM(c *cli.Context, tx *transaction.Transaction) error {
+	var err error
 	if c.IsSet(historicFlagFullName) {
 		height := c.Int(historicFlagFullName)
-		return resetState(c.App, tx, uint32(height))
+		err = resetState(c.App, tx, uint32(height))
+	} else {
+		err = resetState(c.App, tx)
 	}
-
-	return resetState(c.App, tx)
+	if err != nil {
+		return err
+	}
+	if c.IsSet(gasFlagFullName) {
```

Comment:
We have a special `handleLoadDeployed` command, it does magic with `ReuseVM`, so we need to add the same code to the end of `handleLoadDeployed`.

Reply:
Fixed. I've also changed the default behavior of `loadtx` to something probably more expected.